src/test/java/com/snowflake/snowpark_test/JavaUDTFNonStoredProcSuite.java

```diff
@@ -119,6 +119,8 @@ public void testPermanentUdtf() {
       Row[] rows2 = getAnotherSession().tableFunction(permFunc, lit(3), lit(4)).collect();
       assert rows2.length == 1 && rows2[0].size() == 1 && rows2[0].getInt(0) == 7;
 
+      getSession().sql("drop function " + permFunc.funcName() + "(int, int)").collect();
```

Comment:
Is this change related to emit span for udf?

Reply:
no, it is unrelated. I found a bug in this test, the test function hasn't been dropped after test.